RELEASES.md

```diff
@@ -1,3 +1,100 @@
+Version 1.25.0 (2018-03-29)
+==========================
+
+Language
+--------
+- [Stabilised `#[repr(align(x))]`.][47006] [RFC 1358]
+- [You can now use nested groups of imports.][47948]
+  eg. `use std::{fs::File, io::Read, path::{Path, PathBuf}};`
+- [You can now have `|` at the start of a match arm.][47947] eg.
+```rust
+enum Foo { A, B, C }
+
+fn main() {
+    let x = Foo::A;
+    match x {
+        | Foo::A
+        | Foo::B => println!("AB"),
+        | Foo::C => println!("C"),
+    }
+}
+```
+
+Compiler
+--------
+- [Upgraded to LLVM 6.][47828]
+- [Added `-C lto=val` option.][47521]
+- [Added `i586-unknown-linux-musl` target][47282]
+
+Libraries
+---------
+- [Impl Send for `process::Command` on Unix.][47760]
+- [Impl PartialEq and Eq for `ParseCharError`.][47790]
+- [`UnsafeCell::into_inner` is now safe.][47204]
+- [Implement libstd for CloudABI.][47268]
+- [`Float::{from_bits, to_bits}` is now available in libcore.][46931]
+- [renamed `ptr::Shared` to `ptr::NonNull`.][46952]
```

Comment:
this isn't stable yet, so probably shouldn't be in here, right?

Reply:
@steveklabnik Looking at the beta docs and the prs this mentioned in. [It's stable this release](https://doc.rust-lang.org/beta/std/ptr/struct.NonNull.html) unless there was a revert pr I didn't see.

Reply:
yeah, I would make this link to the stabilization of `NonNull` in that case.